### ISSUE TITLE
[Findability] User can't determine where a page is within the VA.gov website.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80589]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User can't determine where a page is within the VA.gov website.
**VA.gov Experience Standard - category:** Findability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA)

### Description

On all pages the current breadcrumb, first segment of title tag before `| Veterans Affairs`, and H1 should be as close to the same text as possible. Currently they're all a little bit different on the Status, Files, and Overview page.

The `Document request` page title, breadcrumb and h1 should use the same text.

All title tags should include `| Veterans Affairs` at the end.

## Recommended action

I understand there has been a lot of dialogue about this! We ask that teams have consistency across these IA waypoints. If all else fails, use the same text for all "Your compensation claim". We have multiple form flows where the title of the form is the title for every page in the flow, so this would be similar.

For `Document request` and `Ask for your claim decision` pages, please include the last part of the title slug, `| Veterans Affairs`, and ensure that the first part of the title tag, current breadcrumb, and h1 are the same text.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Breadcrumbs ](https://design.va.gov/components/breadcrumbs)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Page titles and section titles](https://design.va.gov/content-style-guide/page-titles-and-section-titles)
- **Context:** Navigation
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jerekshoe** [comment]
I merged a PR to update all pages in CST to include `| Veterans Affairs` in all page title tags

**Comment by pmclaren19** [comment]
Follow up questions after eng refinement:

- Currently the page tab text for the Document Request Page is set up to be `Request for ${this.props.trackedItem.displayName}` should we keep this tab text and then update the Document Request Page H1 to follow this text as well? 
![Screenshot 2024-07-02 at 11 08 19 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/af93fb11-7327-41e2-a29f-d15985328a1e)
- If we do the above process suggested then the breadcrumb text would be `Request for ${this.props.trackedItem.displayName}`

**Comment by jacobworrell** [comment]
@uxgary and @skylerschain please determine what the H1 text for the 5103 request detail page should be. Thanks!

**Comment by uxgary** [comment]
@jacobworrell I think we can make the H1 read, "**5103 Evidence Notice**"  since the content below the heading already says "...list the types of evidence we may need..".   

That way we avoid having a breadcrumb that is as long as a sentence.

**Comment by pmclaren19** [comment]
> @jacobworrell I think we can make the H1 read, "**5103 Evidence Notice**" since the content below the heading already says "...list the types of evidence we may need..".
> 
> That way we avoid having a breadcrumb that is as long as a sentence.
 @uxgary @jacobworrell  that sounds fine to me the only thing that I want to bring to ya'lls attention is that the alert that we have that takes a user to this page has the title 'Automated 5103 Notice Response' so are we sure we want a H1 that doesnt match that text? 

EX:
![Screenshot 2024-06-27 at 12 05 10 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/51c927b7-4f0c-4fed-a5a0-53994b4e5866)

**Comment by pmclaren19** [comment]
Talked to @uxgary in stand up and he said thats fine if its different. Will update ac so that the new title is `5103 Evidence Notice` and the breadcrumb matches.

**Comment by skylerschain** [comment]
Yeah this is tricky because although we want plain language H1s, the breadcrumb does need to match the H1 and I agree having a long sentence as a breadcrumb feels strange. I think this decision is fine for now but we might reconsider how we label page tab, breadcrumb, and H1 of the request page when we do our content modeling. cc: @jstrothman  

In the meantime, we should update the 1P alert header so that it matches the H1 on the document request page. I created a ticket for that: https://github.com/department-of-veterans-affairs/va.gov-team/issues/88077

**Comment by pmclaren19** [comment]
@jacobworrell @uxgary verified in staging on user 23 and looks good! Feel free to double verify and let me know if you find any issues! Moving to complete!

**Comment by pmclaren19** [comment]
Hey @jstrothman took a look and looks like this was something that we didn't account for when we initially created the new 5103 page that is displayed on the document request page. 

**Current View of pages:** 
![Screenshot 2024-07-11 at 12 57 05 PM](https://github.com/user-attachments/assets/94d7655b-f816-406c-9153-d56a5cfece7d)
![Screenshot 2024-07-11 at 12 43 29 PM](https://github.com/user-attachments/assets/851e6a59-ad1c-4308-b0eb-1c3845810d84)

The default Document Request page is set up so that the 'Need to mail your files?'  section has a top and bottom margin of 24 px where as the new 5103 Page doesnt have any bottom padding. I can create a quick pr and update this so that the padding is consistent on both pages.

How does this look?

**Updated pages with a bottom margin of 24px:**
![Screenshot 2024-07-15 at 11 52 18 AM](https://github.com/user-attachments/assets/488b562f-902b-42a4-8426-3c23fcda8344)

![Screenshot 2024-07-15 at 11 53 34 AM](https://github.com/user-attachments/assets/f6840eb9-e394-4691-ab0f-a215fbf2c948)

**Comment by jstrothman** [comment]
Thank you, @pmclaren19 The 24px is an improvement, for sure. 
Could you create a backlog issue for adjusting CST-wide to have 80px above the `Need help?` component? 
@jacobworrell I'm not sure how best to get this on the radar for consideration in a future sprint.

**Comment by pmclaren19** [comment]
Created this ticket for the work @jstrothman requested https://github.com/department-of-veterans-affairs/va.gov-team/issues/88588 @jacobworrell .
